tendrl/commons/utils/SSHUtil.py

```diff
@@ -0,0 +1,43 @@
+import psutil
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+def sshd_status():
+    cmd = cmd_utils.Command("cat /var/run/sshd.pid")
```

Comment:
@brainfunked problem is we can't use pip symbol in ansible command module, so grep is not possible